### ISSUE TITLE
So this repository is deprecated, where to find stable Polymer element sets ?

### ISSUE THREAD

**Issue by bcopy** [Polymer/polymer-element-catalog#296]
With the upcoming general availability of https://beta.webcomponents.org/ and the lack of commits on this repository since August 2016, where can we find a stable set and coherent of Polymer elements like was available in the bower.json file of this repository ?
Is there an official replacement with active support ?

@emandi

**Comment by esd100** [comment]
yes...this is a good question. In the past, I would clone the catalog, so that I could browse the information offline if I didn't have an internet connection.

**Comment by arthurevans** [comment]
As far as cloning for offline browsing goes, you can find the new catalog here: https://github.com/webcomponents/beta
